tests/Unit/Controllers/SprintControllerTest.php

```
@@ -318,6 +318,21 @@ public function testUpdatePreservesExistingName(): void
+        // Regression: router passes $id as string; Sprint::update() requires int.
+        // Without the (int) cast this threw a TypeError.
+        $this->db->method('query')->willReturnOnConsecutiveCalls(
+            $this->stmt($this->sprint),
+            $this->stmt($this->project),
+            $this->stmt(null)
+        );
+        $req = $this->makePostRequest(['name' => 'Updated Sprint', 'team_capacity' => '10']);
+        $this->ctrl($req)->update('1'); // string, not int
+        $this->assertSame('Sprint updated.', $_SESSION['flash_message']);
+        $this->assertSame('/app/sprints?project_id=5', $this->response->redirectedTo);
+    }
```

PHPMD (2.15.0)

[error] testUpdateAcceptsStringIdFromRouter accesses the super-global variable $_SESSION. (undefined)

(Superglobals)